docs/content/direct/wds.md

```diff
@@ -2,68 +2,85 @@
 
 - [What is a WDS?](#what-is-a-wds)
 - [Creating a WDS](#creating-a-wds)
+
   - [Using the KubeFlex CLI](#using-the-kubeflex-cli)
-- [KubeFlex Hosting Cluster as WDS](#kubeflex-hosting-cluster-as-wds)
+  - [Accessing the WDS](#accessing-the-wds)
+
+- [Working with a WDS](#working-with-a-wds)
 - [WDS vs. ControlPlane Registration](#wds-vs-controlplane-registration)
 - [Controllers Running in a WDS](#controllers-running-in-a-wds)
-- [Working with a WDS](#working-with-a-wds)
-  - [Accessing the WDS](#accessing-the-wds)
 
-A Workload Description Space (WDS) is a core component of the KubeStellar architecture that serves as the primary interface for users to define and manage workloads for multi-cluster deployment.
+
 
 ## What is a WDS?
 
-A WDS is a space (a Kubernetes-like API server with storage) that:
+A Workload Description Space (WDS) is a space in the [KubeStellar architecture](architecture.md) that serves as the primary interface for users to define and manage workloads for multi-cluster deployment. The WDS constitue of a Kubernetes API server with storage that:
```

Comment:
If I understand your intent correctly, you want to use the word "consists" rather than "constitute".